app/queries/get_edition_for_content_store.rb

```diff
@@ -0,0 +1,18 @@
+module Queries
+  class GetEditionForContentStore
+    def self.call(content_id, locale, include_draft = false)
+      allowed_states = [:published, :unpublished]
+      allowed_states << :draft if include_draft
+
+      scope = Edition.with_document
+      scope = Unpublishing.join_editions(scope)
```

Comment:
We tried this originally, but Rails wasn't having it because the foreign key relation is not defined in the database. We could create a migration to add this constraint though, what do you think?